apps/synapse-oauth-signin/package.json

```diff
@@ -10,7 +10,7 @@
     "@mui/material": "^5.14.17",
     "@react-hookz/web": "^23.1.0",
     "@sage-bionetworks/react-base-table": "^1.13.4",
-    "history": "^5.3.0",
+    "history": "^4.10.1",
```

Comment:
I wish we could add a comment in json to describe the situation.  What do you think about opening a ticket to upgrade react-router to v6, and bump history back to v5?

Reply:
I added a comment to PORTALS-2208